internal/replication/blockreplicator.go

```diff
@@ -309,8 +309,11 @@ func (br *BlockReplicator) deliverEntries(committedEntries []raftpb.Entry) bool
 			}
 
 			if reConfig != nil {
-				// TODO support reconfig
-				br.lg.Panic("Unexpected update to ClusterConfig during normal entry")
+				clusterConfig := reConfig.(*types.ClusterConfig)
+				if err := br.updateClusterConfig(clusterConfig); err != nil {
+					// TODO support dynamic re-config
+					br.lg.Panicf("Unexpected update to ClusterConfig during normal entry: error: %s", err)
```

Comment:
Can you rephrase log message?

Reply:
I hope this is better"
`br.lg.Panicf("Failed to update to ClusterConfig during raft normal entry: error: %s", err)`

Anyway, this log message is temporary until dynamic consensus config is implemented...